src/main/java/fr/openent/viescolaire/model/InitForm/InitFormTimetable.java

```diff
@@ -0,0 +1,120 @@
+package fr.openent.viescolaire.model.InitForm;
+import fr.openent.viescolaire.core.constants.*;
+import fr.openent.viescolaire.core.enums.*;
+import fr.openent.viescolaire.model.IModel;
+import fr.openent.viescolaire.model.SlotProfile.*;
+import fr.openent.viescolaire.utils.*;
+import io.vertx.core.json.*;
+import io.vertx.core.logging.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class InitFormTimetable implements IModel<InitFormTimetable> {
+
+    private static Logger log =  LoggerFactory.getLogger(InitFormTimetable.class);
+
+    private JsonObject morning;
+    private JsonObject afternoon;
+    private List<DayOfWeek> fullDays;
+    private List<DayOfWeek> halfDays;
+
+    public InitFormTimetable() {
+    }
+
+    public InitFormTimetable(JsonObject json) {
+        this.morning = json.getJsonObject(Field.MORNING);
+        this.afternoon = json.getJsonObject(Field.AFTERNOON);
+        this.fullDays = json.getJsonArray(Field.FULLDAYS).stream()
+                .map(o -> DayOfWeek.valueOf((String) o)).collect(Collectors.toList());
```

Comment:
Attention au IllegalArgumentException (No enum constant)
```suggestion
        this.fullDays = json.getJsonArray(Field.FULLDAYS).stream()
                .filter(String.class::isInstance)
                .map(String.class::cast)
                .map(fullDaysValue -> DayOfWeek.getValue(fullDaysValue) //voir comm dans DayOfWeek
                .filter(Objects::notNull)
                .collect(Collectors.toList());
```